e2e/libs/keywords/volume_keywords.py

```diff
@@ -159,6 +161,62 @@ def wait_for_replica_rebuilding_to_complete(self, volume_name):
             logging(f"Waiting for volume {volume_name}'s replica on node {node_id} rebuilding completed")
             self.volume.wait_for_replica_rebuilding_complete(volume_name, node_id)
 
+    async def only_one_replica_rebuilding_will_start_at_a_time_on_node(self, volume_name_0, volume_name_1, replica_locality):
+
+        node_id = self.get_node_id_by_replica_locality(volume_name_0, replica_locality)
+
+        first_replica_rebuilding = None
+        not_start_replica_rebuilding = None
+
+        async def find_first_replica_rebuilding():
+            tasks = [
+                asyncio.create_task(self.volume.wait_for_replica_rebuilding_start(volume_name_0, node_id), name=volume_name_0),
+                asyncio.create_task(self.volume.wait_for_replica_rebuilding_start(volume_name_1, node_id), name=volume_name_1)
+            ]
+
+            done, pending = await asyncio.wait(tasks, return_when=asyncio.FIRST_COMPLETED)
+            for pending_task in pending:
+                pending_task.cancel()
+            return done.pop().get_name(), pending.pop().get_name()
+
+        first_replica_rebuilding, not_start_replica_rebuilding = await find_first_replica_rebuilding()
+        logging(f"Observed {first_replica_rebuilding} started replica rebuilding first")
+
+        while self.volume.is_replica_rebuilding_in_progress(first_replica_rebuilding, node_id):
+            logging(f"Checking volume {not_start_replica_rebuilding} replica rebuilding won't start \
+                if volume {first_replica_rebuilding} replica rebuilding is still in progress")
+            assert not self.volume.is_replica_rebuilding_in_progress(not_start_replica_rebuilding, node_id)
+            time.sleep(1)
+
+    async def both_replica_rebuildings_will_start_at_the_same_time_on_node(self, volume_name_0, volume_name_1, replica_locality):
```

Comment:
Hi @yangchiu 
       Do you think it's necessary to add a `timeout` to ensure that the `both_replica_rebuildings_will_start_at_the_same_time_on_node` &  `only_one_replica_rebuilding_will_start_at_a_time_on_node` doesn't get stuck indefinitely?

Reply:
Each underlay function has its own timeout, like the retry_count in `wait_for_replica_rebuilding_start`. Does it make sense to you?

Reply:
> Each underlay function has its own timeout, like the retry_count in `wait_for_replica_rebuilding_start`. Does it make sense to you?

LGTM